### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mocnik-science**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+
 # GeocodingTools
 
-Incorporates Franz-Benjamin Mocnik's [OSMPythonTools](https://github.com/mocnik-science/osm-python-tools) library into a more general library for geocoding using supported gazetteer services.
-Currently supported are: OpenStreetMap [Nominatim](http://nominatim.openstreetmap.org), [Geonames](http://www.geonames.org/). Endpoint implementations rely on the internal `CacheObject` to comply
-with API rate limits and obligations of clients to cache requests.
+This fork, in the following referred to as `GeocodingTools`, **heavily** builds on Franz-Benjamin Mocnik's [OSMPythonTools](https://github.com/mocnik-science/osm-python-tools) library. In my role as the maintainer of this fork, I have extended the original code and documentation to allow querying the Geonames service.
+
+Currently, the following gazetter services are supported: OpenStreetMap [Nominatim](http://nominatim.openstreetmap.org), [Geonames](http://www.geonames.org/). I aim to support additional gazetteer services in the future. Endpoint implementations rely on the internal `CacheObject` to comply with API rate limits and obligations of clients to cache requests.
```
Comment (ad2476) [inline README.md:9]:
```suggestion
Currently, the following gazetter services are supported: OpenStreetMap [Nominatim](http://nominatim.openstreetmap.org), [Geonames](http://www.geonames.org/). Endpoint implementations rely on the internal `CacheObject` to comply with API rate limits and obligations of clients to cache requests.
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
+# About This Fork
+
+**This library is an unofficial fork of [OSMPythonTools](https://github.com/mocnik-science/osm-python-tools). If you are looking for the original `OSMPythonTools` library, please refer to [https://github.com/mocnik-science/osm-python-tools](https://github.com/mocnik-science/osm-python-tools).**
+
 # GeocodingTools
 
-Incorporates Franz-Benjamin Mocnik's [OSMPythonTools](https://github.com/mocnik-science/osm-python-tools) library into a more general library for geocoding using supported gazetteer services.
-Currently supported are: OpenStreetMap [Nominatim](http://nominatim.openstreetmap.org), [Geonames](http://www.geonames.org/). Endpoint implementations rely on the internal `CacheObject` to comply
-with API rate limits and obligations of clients to cache requests.
+This fork, in the following referred to as `GeocodingTools`, **heavily** builds on Franz-Benjamin Mocnik's [OSMPythonTools](https://github.com/mocnik-science/osm-python-tools) library. In my role as the maintainer of this fork, I have extended the original code and documentation to allow querying the Geonames service.
```
Comment (ad2476) [inline README.md:7]:
```suggestion
This fork, heavily builds on Franz-Benjamin Mocnik's [OSMPythonTools](https://github.com/mocnik-science/osm-python-tools) library. I have extended the original code and documentation to allow querying the Geonames service.
```

**Review by ad2476 — CHANGES_REQUESTED** [review]
This fork has been mostly dead, so I'd not like to promise future development. If it's caused confusion about the original, happy to merge this PR to clarify.

File: README.md (hunk 2 above)
Comment (mocnik-science) [reply README.md:7]: fine with it.

Comment (mocnik-science) [reply README.md:7]: Thanks

**Comment by mocnik-science** [comment]
Feel welcome to merge.

**Comment by mocnik-science** [comment]
Thanks for your quick reaction.

(And please do not forget to adapt the "About" message like proposed by me.  Thanks!

**Pull request merged**